content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -279,38 +279,26 @@ When using `systemctl` to run .NET applications as a service, you can also set e
 
 {{< /tabs >}}
 
-## Custom instrumentation
+### Configure the Datadog Agent for APM
```

Comment:
The diff looks confusing but I moved the "Configure the Datadog Agent for APM" section so it is now the last subsection of "Installation and getting started" and I also updated the text to match the other languages